util/crypter.js

```diff
@@ -1,5 +1,7 @@
-const crypto = require('crypto');
+var dotenv = require('dotenv');
+dotenv.config({ path: './.env'});
 
```

Comment:
export the environment in test.js instead of importing environmental variable in crypter.js will look good.